### DISCUSSION TITLE
File Finder: show previous buffer at top instead of current buffer when toggling

### DISCUSSION THREAD

**Discussion by mkdynamic** [zed-industries/zed#6598] [Feature Requests]
In Sublime Text file finder there is subtle a behavior which makes bouncing between files very convenient:

1. Open the File finder panel, jump to a file (A)
2. Do some stuff in that file (A)
3. Open the File finder panel, jump to another file (B)
4. Do some stuff in that file (B)
5. Open the File finder panel, it shows file (A) at the top
6. Jump back to file (A)
7. Open the File finder panel, it shows file (B) at the top
8. Jump back to file (B)
9. Etc.

I'm not actually sure of the algorithm that is used (some kind of cycling list I think), but switching from Sublime to Zed made me notice the different behavior. It may be an idiosyncratic thing I rely on, but for me was a behavior I relied on heavily to navigate between multiple files.

Would love to see this in Zed.

**Comment by mkdynamic** [comment]
For me, this is solved by `0.122.2` 🙌

**Reply by boxxroom** [reply]
@mkdynamic Working with Sublime Text for ver 10+ years, I also used this very heavily. The quick switch between A / B files really speeds up production. 

TLDR; You mention this was solved in `0.122.2`. How has it been fixed

I am using `0.230.1` and feel the `File Finder` still behaves slightly different. In that, the `File Finder` appears to display a `history of files` rather than your `open files` while switching between files require `cmd+p` + `p` to navigate down to the next file then release. In Sublime Text (as you know) `cmd+p` automatically selects the next file down and you release. This small behaviour speeds up working with files considerably.
